packages/presentation/presentation_containers/interfaces/presentation_container.ts

```diff
@@ -13,13 +13,16 @@ import {
   PublishingSubject,
 } from '@kbn/presentation-publishing';
 import { apiCanAddNewPanel, CanAddNewPanel } from './can_add_new_panel';
+import { PublishesSettings } from './publishes_settings';
 
 export interface PanelPackage {
   panelType: string;
   initialState?: object;
 }
 
-export interface PresentationContainer extends Partial<PublishesViewMode>, CanAddNewPanel {
+export interface PresentationContainer
+  extends Partial<PublishesViewMode & PublishesSettings>,
+    CanAddNewPanel {
   addNewPanel: <ApiType extends unknown = unknown>(
```

Comment:
I just noticed that this `addNewPanel` gets redefined in here. Would you mind removing it? Must have snuck in during a merge on my side.

Reply:
done in 5bcfb4aae842aeb563836e170abd8946671e687c